cdk/lib/registration.ts

```diff
@@ -17,43 +14,8 @@ export class Registration extends GuStack {
 		// Until this project has been fully migrated to GuCDK you should update the 'old' infrastructure by modifying
 		// the YAML file and then re-running the snapshot tests to confirm that the changes are being pulled through by
 		// CDK
-		const cfnTemplate = new CfnInclude(this, 'YamlTemplate', {
+		new CfnInclude(this, 'YamlTemplate', {
 			templateFile: yamlTemplateFilePath,
 		});
-
-		const registrationLoadBalancer = cfnTemplate.getResource(
-			'LoadBalancerToPrivateASG',
-		) as CfnLoadBalancer;
-
-		const loadBalancerName = registrationLoadBalancer.ref;
-
-		function loadBalancerMetric(metricName: string) {
-			return new Metric({
-				metricName,
-				namespace: 'AWS/ELB',
-				statistic: 'Sum',
-				dimensionsMap: {
-					LoadBalancerName: loadBalancerName,
-				},
-			});
-		}
-
-		const applicationServerErrors = loadBalancerMetric('HTTPCode_Backend_5XX');
-		const loadBalancerErrors = loadBalancerMetric('HTTPCode_ELB_5XX');
-		const allRequests = loadBalancerMetric('RequestCount');
-
-		const budgetAlarm = new GuErrorBudgetAlarmExperimental(this, {
-			sloName: `RegistrationAvailability${this.stage}Slo`,
-			sloTarget: 0.995,
-			badEvents: new MathExpression({
```

Comment:
We can generalise the `badEvents` and `validEvents` needed for a 'typical' Availability SLI to work for every load balancer - so this now happens via some code that @guardian/devx-reliability will maintain: https://github.com/guardian/slo-alerts/blob/f715cb3c6cb48d5885d2ebe5d987f85acae4547b/cdk/lib/slo-alerts.ts#L28-L92